certbot/plugins/manual.py

```diff
@@ -42,7 +42,8 @@ class Authenticator(common.Plugin):
 
 {validation}
 
-Once this is deployed,"""
+Once this TXT record is deployed,
+Let's Encrypt will be able to check the validity of your domain."""
```

Comment:
@bmw Thanks for reviewing this PR!  I updated the wording for a gentle alert to take action before proceeding.  What do you think?